src/modules/Calendar/CalendarNavBar.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {connect} from "react-redux";
+import {monthsShort} from "moment";
+
+import '../../styles/CalendarStyles/CalendarNavBar.css'
+import {getSelectedMonth, getSelectedYear} from './_selectors'
+import {selectMonth, selectYear} from './_actionsDefinitions';
+
+
+class CalendarNavBar extends React.Component {
+
+    selectMonthOnClick(index) {
```

Comment:
does it work without binding? better to use arrow function here to avoid possible crashes in the future